packages/components/src/tooltip/types.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import type * as Ariakit from '@ariakit/react/tooltip';
+/**
+ * Internal dependencies
+ */
+import type { PopoverProps } from '../popover/types';
+import type { ShortcutProps } from '../shortcut/types';
+
+export type TooltipProps = {
+	/**
+	 * The anchor for the tooltip.
+	 *
+	 * **Note**: Accepts only one child element.
+	 */
+	children: Ariakit.TooltipProps[ 'render' ];
```

Comment:
When I add more than one child, I do not see a type error, but we get one in our test for it:

https://github.com/WordPress/gutenberg/blob/cf59cd437ac97551491af8228ee9aa8563e2870d/packages/components/src/tooltip/test/index.tsx#L26-L29

I'm not sure why that is. @chad1008 or @ciampo, do you have any insight here?

Reply:
> we get one in our test for it

The one that we get in our tests is working as expected (the `@ts-expect-error` is correct).

Where else have you tried adding more than one child? Can you share an example?

Reply:
>Where else have you tried adding more than one child? Can you share an example?

Ignore me! I just realized it was in a `.js` file 🤦‍♀️